### ISSUE TITLE
Libunistring build fails when gettext installed

### ISSUE THREAD

**Issue by hutson** [Homebrew/linuxbrew-core#21795]
# Bug report

- [x] ran `brew update` and can still reproduce the problem?
- [ ] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

## What you were trying to do (and why)

I was attempting to install `git` from source (As I use a non-standard Homebrew directory).

```
brew install git
```

## What happened (include command output)

The `libunistring` formula, a dependency of `git`, fails to compile. This failure seems related to `gettext`, another dependency of `git`, which is installed just prior to `libunistring` getting compiled:

```
==> Installing dependencies for git: gettext, libunistring
```

<details>
  <summary>Command output</summary>
  <pre>

==> Installing dependencies for git: gettext, libunistring, libidn2, libbsd, expat, libmetalink, libssh2, c-ares, jemalloc, libev, nghttp2, berkeley-db, perl, texinfo, jpeg, libtiff, libpng, freetype, gperf, json-c, util-linux, fontconfig, libidn, ghostscript, util-macros, xtrans, xcb-proto, libpthread-stubs, xorgproto, libxau, libxdmcp, libxcb, libx11, libxext, libxfixes, libxi, libxrender, libxrandr, libxxf86vm, libxdamage, bison, help2man, flex, libedit, gmp, mpfr, libmpc, binutils, isl@0.18, gcc, autoconf, automake, libelf, llvm, lm-sensors, libxshmfence, libxv, libxvmc, docbook, docbook-xsl, libxslt, libpciaccess, libdrm, linuxbrew/xorg/libomxil-bellagio, linuxbrew/xorg/wayland, linuxbrew/xorg/libva-internal, linuxbrew/xorg/libvdpau, linuxbrew/xorg/wayland-protocols, mesa, mesa-glu, libxinerama, xinput, freeglut, jasper, netpbm, psutils, uchardet, glib, groff, openldap, rtmpdump, zstd, curl and linux-headers
==> Installing git dependency: gettext
==> ./configure --prefix=/home/hutson/.linuxbrew/Cellar/gettext/0.21_1 --with-included-gettext --with-included-glib --with-inclu
==> make
==> make install
🍺  /home/hutson/.linuxbrew/Cellar/gettext/0.21_1: 1,957 files, 21.8MB, built in 7 minutes 26 seconds
==> Installing git dependency: libunistring
==> ./configure --prefix=/home/hutson/.linuxbrew/Cellar/libunistring/0.9.10
==> make
==> make check
Last 15 lines from /home/hutson/.cache/Homebrew/Logs/libunistring/03.make:
# ERROR: 0
============================================================================
See tests/test-suite.log
============================================================================
make[5]: *** [Makefile:10119: test-suite.log] Error 1
make[5]: Leaving directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
make[4]: *** [Makefile:10227: check-TESTS] Error 2
make[4]: Leaving directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
make[3]: *** [Makefile:13900: check-am] Error 2
make[3]: Leaving directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
make[2]: *** [Makefile:10011: check-recursive] Error 1
make[2]: Leaving directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
make[1]: *** [Makefile:13902: check] Error 2
make[1]: Leaving directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
make: *** [Makefile:1559: check-recursive] Error 1

READ THIS: https://docs.brew.sh/Troubleshooting

  </pre>
</details>

<details>
<pre>
make[5]: Entering directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
SKIP: test-btowc1.sh
SKIP: test-btowc2.sh
PASS: test-ctype
PASS: test-alloca-opt
PASS: test-environ
PASS: test-c-ctype
PASS: test-array-mergesort
PASS: test-errno
PASS: test-fdopen
PASS: test-c-strcase.sh
PASS: test-float
PASS: test-fputc
PASS: test-fgetc
PASS: test-frexp-nolibm
PASS: test-fread
PASS: test-fseterr
PASS: test-frexpl-nolibm
PASS: test-fwrite
PASS: test-iconv-h
PASS: test-iconv
PASS: test-getprogname
PASS: test-intprops
PASS: test-inttypes
PASS: test-isblank
PASS: test-isnand-nolibm
PASS: test-isnanf-nolibm
PASS: test-isnanl-nolibm
PASS: test-langinfo
PASS: test-iswblank
PASS: test-limits-h
PASS: test-locale
PASS: test-localename
SKIP: test-mbrtowc1.sh
SKIP: test-mbrtowc2.sh
PASS: test-once1
PASS: test-once2
SKIP: test-mbrtowc3.sh
SKIP: test-mbrtowc4.sh
PASS: test-math
PASS: test-malloca
SKIP: test-mbrtowc-w32-1.sh
SKIP: test-mbrtowc-w32-3.sh
SKIP: test-mbsinit.sh
SKIP: test-mbrtowc-w32-4.sh
SKIP: test-mbrtowc-w32-2.sh
PASS: test-mbrtowc5.sh
PASS: test-printf-frexp
SKIP: test-mbrtowc-w32-5.sh
PASS: test-printf-frexpl
SKIP: test-setlocale1.sh
PASS: test-rwlock1
PASS: test-raise
PASS: test-setenv
PASS: test-signal-h
PASS: test-signbit
PASS: test-stdalign
PASS: test-stdbool
PASS: test-memchr
PASS: test-stddef
PASS: test-stdint
PASS: test-stdio
PASS: test-stdlib
PASS: test-strerror
PASS: test-striconveh
PASS: test-setlocale2.sh
PASS: test-striconveha
PASS: test-string
PASS: test-strncat
PASS: test-sys_types
PASS: test-thread_self
PASS: test-init.sh
FAIL: test-thread_create
PASS: test-strstr
PASS: test-time
PASS: test-cased
PASS: test-ignorable
PASS: unicase/test-locale-language.sh
PASS: test-uc_tolower
PASS: test-uc_totitle
PASS: test-u16-casecmp
PASS: test-uc_toupper
PASS: test-u16-casecoll
PASS: test-u16-casefold
PASS: test-u16-is-casefolded
PASS: test-u16-is-cased
PASS: test-u16-is-titlecase
PASS: test-u16-is-lowercase
PASS: test-u16-is-uppercase
PASS: test-u16-tolower
PASS: test-u16-totitle
PASS: test-u16-toupper
PASS: test-u32-casecoll
PASS: test-u32-casefold
PASS: test-u32-casecmp
PASS: test-u32-is-cased
PASS: test-u32-is-casefolded
PASS: test-u32-is-lowercase
PASS: test-u32-is-uppercase
PASS: test-u32-is-titlecase
PASS: test-u32-tolower
PASS: test-u32-totitle
PASS: test-u32-toupper
PASS: test-u8-casecmp
PASS: test-u8-casecoll
PASS: test-u8-casefold
PASS: test-u8-is-cased
PASS: test-u8-is-casefolded
PASS: test-u8-is-lowercase
PASS: test-u8-is-titlecase
PASS: test-u8-is-uppercase
PASS: test-u8-tolower
PASS: test-u8-totitle
PASS: test-u8-toupper
SKIP: unicase/test-ulc-casecmp1.sh
SKIP: unicase/test-ulc-casecmp2.sh
SKIP: unicase/test-ulc-casecoll1.sh
SKIP: unicase/test-ulc-casecoll2.sh
PASS: test-pthread_sigmask2
PASS: test-pthread_sigmask1
PASS: test-u16-conv-from-enc
PASS: test-u16-strconv-to-enc
PASS: test-u32-conv-from-enc
PASS: test-u16-strconv-from-enc
PASS: test-u16-conv-to-enc
PASS: test-sigprocmask
PASS: test-u32-strconv-from-enc
PASS: test-u32-conv-to-enc
PASS: test-sleep
PASS: test-u8-conv-to-enc
PASS: test-u8-conv-from-enc
PASS: test-u8-strconv-from-enc
PASS: test-bidi_longname
PASS: test-u8-strconv-to-enc
PASS: test-u32-strconv-to-enc
PASS: test-bidi_byname
PASS: test-bidi_name
PASS: test-block_list
PASS: test-bidi_of
PASS: test-block_test
PASS: test-bidi_test
PASS: test-categ_C
PASS: test-block_of
PASS: test-categ_Cc
PASS: test-categ_Cf
PASS: test-categ_Co
PASS: test-categ_Cn
PASS: test-categ_Cs
PASS: test-categ_LC
PASS: test-categ_L
PASS: test-categ_Ll
PASS: test-categ_Lm
PASS: test-categ_Lo
PASS: test-categ_Lt
PASS: test-categ_M
PASS: test-categ_Lu
PASS: test-categ_Mc
PASS: test-categ_Me
PASS: test-categ_Mn
PASS: test-categ_N
PASS: test-categ_Nd
PASS: test-categ_No
PASS: test-categ_Nl
PASS: test-categ_P
PASS: test-categ_Pc
PASS: test-categ_Pd
PASS: test-categ_Pe
PASS: test-categ_Pf
PASS: test-categ_Po
PASS: test-categ_S
PASS: test-categ_Pi
PASS: test-categ_Ps
PASS: test-categ_Sk
PASS: test-categ_Sm
PASS: test-categ_So
PASS: test-categ_Sc
PASS: test-categ_Zl
PASS: test-categ_Zp
PASS: test-categ_Z
PASS: test-categ_and
PASS: test-categ_and_not
PASS: test-categ_Zs
PASS: test-categ_longname
PASS: test-categ_name
PASS: test-categ_byname
PASS: test-categ_none
PASS: test-categ_or
PASS: test-categ_of
PASS: test-categ_test_withtable
PASS: test-combiningclass_longname
PASS: test-combiningclass_name
PASS: test-combiningclass_byname
PASS: test-combiningclass
PASS: test-ctype_alnum
PASS: test-ctype_alpha
PASS: test-ctype_blank
PASS: test-ctype_cntrl
PASS: test-ctype_graph
PASS: test-ctype_digit
PASS: test-ctype_lower
PASS: test-ctype_print
PASS: test-ctype_punct
PASS: test-ctype_space
PASS: test-ctype_upper
PASS: test-ctype_xdigit
PASS: test-digit
PASS: test-decdigit
PASS: test-joininggroup_byname
PASS: test-joininggroup_name
PASS: test-joininggroup_of
PASS: test-joiningtype_longname
PASS: test-joiningtype_byname
PASS: test-joiningtype_name
PASS: test-mirror
PASS: test-numeric
PASS: test-joiningtype_of
PASS: test-pr_alphabetic
PASS: test-pr_bidi_arabic_right_to_left
PASS: test-pr_ascii_hex_digit
PASS: test-pr_bidi_arabic_digit
PASS: test-pr_bidi_boundary_neutral
PASS: test-pr_bidi_block_separator
PASS: test-pr_bidi_control
PASS: test-pr_bidi_common_separator
PASS: test-pr_bidi_embedding_or_override
PASS: test-pr_bidi_eur_num_separator
PASS: test-pr_bidi_eur_num_terminator
PASS: test-pr_bidi_hebrew_right_to_left
PASS: test-pr_bidi_european_digit
PASS: test-pr_bidi_non_spacing_mark
PASS: test-pr_bidi_left_to_right
PASS: test-pr_bidi_other_neutral
PASS: test-pr_bidi_pdf
PASS: test-pr_bidi_segment_separator
PASS: test-pr_bidi_whitespace
PASS: test-pr_cased
PASS: test-pr_case_ignorable
PASS: test-pr_changes_when_casefolded
PASS: test-pr_byname
PASS: test-pr_changes_when_casemapped
PASS: test-pr_changes_when_titlecased
PASS: test-pr_changes_when_lowercased
PASS: test-pr_combining
PASS: test-pr_changes_when_uppercased
PASS: test-pr_composite
PASS: test-pr_currency_symbol
PASS: test-pr_dash
PASS: test-pr_decimal_digit
PASS: test-pr_deprecated
PASS: test-pr_default_ignorable_code_point
PASS: test-pr_diacritic
PASS: test-pr_extender
PASS: test-pr_format_control
PASS: test-pr_grapheme_base
PASS: test-pr_hex_digit
PASS: test-pr_grapheme_link
PASS: test-pr_grapheme_extend
PASS: test-pr_hyphen
PASS: test-pr_id_continue
PASS: test-pr_id_start
PASS: test-pr_ideographic
PASS: test-pr_ignorable_control
PASS: test-pr_ids_trinary_operator
PASS: test-pr_ids_binary_operator
PASS: test-pr_join_control
PASS: test-pr_iso_control
PASS: test-pr_left_of_pair
PASS: test-pr_line_separator
PASS: test-pr_logical_order_exception
PASS: test-pr_lowercase
PASS: test-pr_math
PASS: test-pr_non_break
PASS: test-pr_not_a_character
PASS: test-pr_numeric
PASS: test-pr_other_default_ignorable_code_point
PASS: test-pr_other_alphabetic
PASS: test-pr_other_id_continue
PASS: test-pr_other_grapheme_extend
PASS: test-pr_other_lowercase
PASS: test-pr_other_id_start
PASS: test-pr_other_math
PASS: test-pr_other_uppercase
PASS: test-pr_paired_punctuation
PASS: test-pr_paragraph_separator
PASS: test-pr_pattern_syntax
PASS: test-pr_pattern_white_space
PASS: test-pr_private_use
PASS: test-pr_punctuation
PASS: test-pr_quotation_mark
PASS: test-pr_radical
PASS: test-pr_sentence_terminal
PASS: test-pr_soft_dotted
PASS: test-pr_terminal_punctuation
PASS: test-pr_space
PASS: test-pr_test
PASS: test-pr_unassigned_code_value
PASS: test-pr_titlecase
PASS: test-pr_variation_selector
PASS: test-pr_white_space
PASS: test-pr_unified_ideograph
PASS: test-pr_uppercase
PASS: test-pr_xid_continue
PASS: test-pr_zero_width
PASS: test-pr_xid_start
PASS: test-sy_c_ident
PASS: test-scripts
PASS: test-sy_java_ident
PASS: test-sy_c_whitespace
PASS: test-u16-grapheme-breaks
PASS: test-u16-grapheme-next
PASS: test-sy_java_whitespace
PASS: test-u32-grapheme-next
PASS: test-u32-grapheme-breaks
PASS: test-u16-grapheme-prev
PASS: test-u32-grapheme-prev
PASS: test-u8-grapheme-next
PASS: test-u8-grapheme-breaks
PASS: test-u8-grapheme-prev
SKIP: unigbrk/test-ulc-grapheme-breaks.sh
PASS: test-uc-gbrk-prop
PASS: unigbrk/test-uc-grapheme-breaks.sh
PASS: test-u16-width-linebreaks
PASS: unigbrk/test-uc-is-grapheme-break.sh
PASS: test-u16-possible-linebreaks
PASS: test-u32-possible-linebreaks
PASS: test-u32-width-linebreaks
PASS: test-u8-possible-linebreaks
PASS: test-u8-width-linebreaks
PASS: test-ulc-width-linebreaks
PASS: test-ulc-possible-linebreaks
PASS: test-canonical-decomposition
PASS: test-composition
PASS: test-compat-decomposition
PASS: test-decomposition
PASS: test-uninorm-filter-nfc
PASS: test-decomposing-form
PASS: test-nfd
PASS: uninorm/test-u32-nfc-big.sh
PASS: test-nfc
PASS: uninorm/test-u32-nfkc-big.sh
PASS: uninorm/test-u32-nfd-big.sh
PASS: test-u16-normcmp
PASS: test-nfkd
PASS: test-u16-normcoll
PASS: test-nfkc
PASS: test-u32-normcmp
PASS: test-u32-normcoll
PASS: test-unistd
PASS: test-u8-normcoll
PASS: test-u8-normcmp
SKIP: unistdio/test-u16-vasnprintf2.sh
SKIP: unistdio/test-u16-vasnprintf3.sh
PASS: uninorm/test-u32-nfkd-big.sh
PASS: test-u16-asnprintf1
PASS: test-u16-vasnprintf1
PASS: test-u16-vasprintf1
PASS: uniname/test-uninames.sh
PASS: test-u16-vsnprintf1
PASS: test-u32-asnprintf1
SKIP: unistdio/test-u32-vasnprintf2.sh
PASS: test-u16-vsprintf1
PASS: test-u32-vasnprintf1
SKIP: unistdio/test-u32-vasnprintf3.sh
SKIP: unistdio/test-u8-vasnprintf2.sh
PASS: test-u32-vasprintf1
SKIP: unistdio/test-u8-vasnprintf3.sh
PASS: test-u32-vsprintf1
PASS: test-u8-vasnprintf1
PASS: test-u32-vsnprintf1
PASS: test-u8-asnprintf1
SKIP: unistdio/test-ulc-vasnprintf3.sh
SKIP: unistdio/test-ulc-vasnprintf2.sh
PASS: test-u8-vasprintf1
PASS: test-u8-vsnprintf1
PASS: test-u8-vsprintf1
PASS: test-ulc-vasnprintf1
PASS: test-ulc-asnprintf1
PASS: test-u16-check
PASS: test-ulc-vasprintf1
PASS: test-ulc-vsnprintf1
PASS: test-ulc-vsprintf1
PASS: test-u16-cmp2
PASS: test-u16-cmp
PASS: test-u16-mbsnlen
PASS: test-u16-cpy
PASS: test-u16-cpy-alloc
PASS: test-u16-mblen
PASS: test-u16-chr
PASS: test-u16-mbtouc
PASS: test-u16-mbtouc-unsafe
PASS: test-u16-mbtoucr
PASS: test-u16-move
PASS: test-u16-next
PASS: test-u16-prev
PASS: test-u16-set
PASS: test-u16-stpcpy
PASS: test-u16-strcat
PASS: test-u16-stpncpy
PASS: test-u16-strcmp
PASS: test-u16-strcoll
PASS: test-u16-strcpy
PASS: test-u16-strdup
PASS: test-u16-strchr
PASS: test-u16-strlen
PASS: test-u16-strmbtouc
PASS: test-u16-strmblen
PASS: test-u16-strncmp
PASS: test-u16-strncat
PASS: test-u16-strnlen
PASS: test-u16-strncpy
PASS: test-u16-strtok
PASS: test-u16-to-u32
PASS: test-u16-to-u8
PASS: test-u16-strstr
PASS: test-u16-uctomb
PASS: test-u32-check
PASS: test-u32-cmp
PASS: test-u32-chr
PASS: test-u32-cmp2
PASS: test-u32-cpy-alloc
PASS: test-u32-mblen
PASS: test-u32-cpy
PASS: test-u32-mbsnlen
PASS: test-u32-mbtouc
PASS: test-u32-mbtoucr
PASS: test-u32-mbtouc-unsafe
PASS: test-u32-move
PASS: test-u32-next
PASS: test-u32-prev
PASS: test-u32-set
PASS: test-u32-stpcpy
PASS: test-u32-stpncpy
PASS: test-u32-strcmp
PASS: test-u32-strcat
PASS: test-u32-strchr
PASS: test-u32-strcoll
PASS: test-u32-strcpy
PASS: test-u32-strdup
PASS: test-u32-strmblen
PASS: test-u32-strlen
PASS: test-u32-strmbtouc
PASS: test-u32-strncat
PASS: test-u32-strncmp
PASS: test-u32-strncpy
PASS: test-u32-strnlen
PASS: test-u32-to-u16
PASS: test-u32-to-u8
PASS: test-u32-strtok
PASS: test-u8-check
PASS: test-u32-strstr
PASS: test-u32-uctomb
PASS: test-u8-cmp
PASS: test-u8-cmp2
PASS: test-u8-cpy-alloc
PASS: test-u8-chr
PASS: test-u8-mblen
PASS: test-u8-mbtouc
PASS: test-u8-cpy
PASS: test-u8-mbsnlen
PASS: test-u8-mbtouc-unsafe
PASS: test-u8-mbtoucr
PASS: test-u8-move
PASS: test-u8-set
PASS: test-u8-next
PASS: test-u8-stpcpy
PASS: test-u8-prev
PASS: test-u8-stpncpy
PASS: test-u8-strcoll
PASS: test-u8-strcat
PASS: test-u8-strcmp
PASS: test-u8-strcpy
PASS: test-u8-strdup
PASS: test-u8-strchr
PASS: test-u8-strlen
PASS: test-u8-strncmp
PASS: test-u8-strmbtouc
PASS: test-u8-strncat
PASS: test-u8-strmblen
PASS: test-u8-strnlen
PASS: test-u8-strncpy
PASS: test-u8-to-u32
PASS: test-u8-strtok
PASS: test-u8-to-u16
PASS: test-u8-uctomb
PASS: test-u16-wordbreaks
PASS: test-u32-wordbreaks
SKIP: uniwbrk/test-ulc-wordbreaks.sh
PASS: uniwbrk/test-uc-wordbreaks.sh
PASS: test-u8-wordbreaks
PASS: test-u8-strstr
PASS: test-u32-strwidth
PASS: test-u16-strwidth
PASS: test-u16-width
PASS: test-u32-width
PASS: test-u8-strwidth
PASS: test-u8-width
PASS: test-uc_width
PASS: test-unsetenv
PASS: test-verify
PASS: test-wcrtomb.sh
PASS: test-wchar
SKIP: test-wcrtomb-w32-1.sh
PASS: uniwidth/test-uc_width2.sh
SKIP: test-wcrtomb-w32-2.sh
SKIP: test-wcrtomb-w32-3.sh
SKIP: test-wcrtomb-w32-4.sh
SKIP: test-wcrtomb-w32-5.sh
PASS: test-wctype-h
PASS: test-wcwidth
PASS: test-xalloc-die.sh
PASS: test-verify.sh
PASS: test-lock
PASS: test-usleep
============================================================================
Testsuite summary for
============================================================================
# TOTAL: 512
# PASS:  479
# SKIP:  32
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See tests/test-suite.log
============================================================================
make[5]: *** [Makefile:10119: test-suite.log] Error 1
make[5]: Leaving directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
make[4]: *** [Makefile:10227: check-TESTS] Error 2
make[4]: Leaving directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
make[3]: *** [Makefile:13900: check-am] Error 2
make[3]: Leaving directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
make[2]: *** [Makefile:10011: check-recursive] Error 1
make[2]: Leaving directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
make[1]: *** [Makefile:13902: check] Error 2
make[1]: Leaving directory '/tmp/libunistring-20201212-1902431-xb7n6n/libunistring-0.9.10/tests'
make: *** [Makefile:1559: check-recursive] Error 1

HOMEBREW_VERSION: 2.6.1
ORIGIN: https://github.com/Homebrew/brew
HEAD: 2c77a540b522c2eee74af0745851167412adb83b
Last commit: 3 days ago
Core tap ORIGIN: https://github.com/Homebrew/linuxbrew-core
Core tap HEAD: e980d226a775052e392ca7b951e838665c42d073
Core tap last commit: 75 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /home/hutson/.linuxbrew
HOMEBREW_REPOSITORY: /home/hutson/.linuxbrew/Homebrew
HOMEBREW_CELLAR: /home/hutson/.linuxbrew/Cellar
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_EDITOR: nvim
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_ANALYTICS: set
Homebrew Ruby: 2.6.3 => /home/hutson/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: octa-core 64-bit skylake
Clang: N/A
Git: 2.25.1 => /bin/git
Curl: 7.68.0 => /usr/bin/curl
Kernel: Linux 5.4.0-58-generic x86_64 GNU/Linux
OS: Ubuntu 20.04.1 LTS (focal)
Host glibc: 2.31
/usr/bin/gcc: 9.3.0
/usr/bin/ruby: N/A
glibc: N/A
gcc: N/A
xorg: N/A

HOMEBREW_CC: gcc-9
HOMEBREW_CXX: g++-9
MAKEFLAGS: -j8
CMAKE_PREFIX_PATH: /home/hutson/.linuxbrew
HOMEBREW_GIT: git
ACLOCAL_PATH: /home/hutson/.linuxbrew/share/aclocal
PATH: /home/hutson/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super:/usr/bin:/bin:/usr/sbin:/sbin
</pre>
</details>

This build has worked for me in the past (As in last year). I suspect the change is related to `gettext` no longer being key-only - Homebrew/homebrew-core#53489

## What you expected to happen

For the `git` build and installation to succeed.

## Step-by-step reproduction instructions (by running `brew install` commands)

```
brew install git
```

If you remove `gettext` and then attempt to install `libunistring`, the `libunistring` build will pass successfully.

```
brew remove gettext
brew install libunistring
```

**Comment by iMichka** [comment]
Can I see your brew config and brew doctor?

**Comment by hutson** [comment]
Here's the information (The config information got lost in the command output details):

```bash
$ brew doctor
brew config
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Your Homebrew's prefix is not /home/linuxbrew/.linuxbrew.
Some of Homebrew's bottles (binary packages) can only be used with the default
prefix (/home/linuxbrew/.linuxbrew).
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
unsupported configuration.

$ brew config
HOMEBREW_VERSION: 2.6.1-86-gced0da1
ORIGIN: https://github.com/Homebrew/brew
HEAD: ced0da159fd06f6d674f4354957a5074735b09aa
Last commit: 18 hours ago
Core tap ORIGIN: https://github.com/Homebrew/linuxbrew-core
Core tap HEAD: c4bdfc797d18c3e93af9e8af040e343eaead9084
Core tap last commit: 4 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /home/hutson/.linuxbrew
HOMEBREW_REPOSITORY: /home/hutson/.linuxbrew/Homebrew
HOMEBREW_CELLAR: /home/hutson/.linuxbrew/Cellar
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_EDITOR: nvim
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_ANALYTICS: set
Homebrew Ruby: 2.6.3 => /home/hutson/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: octa-core 64-bit skylake
Clang: N/A
Git: 2.25.1 => /bin/git
Curl: 7.68.0 => /usr/bin/curl
Kernel: Linux 5.4.0-58-generic x86_64 GNU/Linux
OS: Ubuntu 20.04.1 LTS (focal)
Host glibc: 2.31
/usr/bin/gcc: 9.3.0
/usr/bin/ruby: N/A
glibc: N/A
gcc: N/A
xorg: N/A
```

**Comment by iMichka** [comment]
This should be fixed by #21817. This is a more general fix for all the gettext issues we had in the last days.
